packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0]
+### Changed
+- **BREAKING**: `TokensController.addToken` now accepts a single options object ([#1676](https://github.com/MetaMask/core/pull/1676))
```

Comment:
We should also mention that the `getNetworkClientById` constructor parameter is now required

```suggestion
- **BREAKING**: `TokensController` now expects `getNetworkClientById` in constructor options 
- **BREAKING**: `TokensController.addToken` now accepts a single options object ([#1676](https://github.com/MetaMask/core/pull/1676))
```

This was included in the list of changes in the PR description